quotes.json

```
@@ -3783,6 +3783,10 @@
     "author": "Mahatma Gandhi",
     "text": "We must become the change we want to see."
   },
+  {
+    "author": "Mahatma Gandhi",
+    "text": "Live as if you were to die tomorrow.Learn as if you were to live forever."
```

"tomorrow.Learn" >> "tomorrow. Learn" ...
```suggestion
"text": "Live as if you were to die tomorrow. Learn as if you were to live forever."
```
Just adding a space between the full stop and "Learn".